OnlineGameStore.Application/Services/Implementation/UserService.cs

```diff
@@ -0,0 +1,79 @@
+﻿using MapsterMapper;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using OnlineGameStore.Application.Auth;
+using OnlineGameStore.Application.Exceptions;
+using OnlineGameStore.Application.Models.Requests;
+using OnlineGameStore.Application.Models.Views;
+using OnlineGameStore.Application.Services.Constants;
+using OnlineGameStore.Application.Services.Interfaces;
+using OnlineGameStore.Infrastructure.Identity;
+
+namespace OnlineGameStore.Application.Services.Implementation
+{
+    public class UserService : IUserService
+    {
+        private readonly UserManager<User> _userManager;
+        private readonly IMapper _mapper;
+        private readonly ICurrentUser _currentUser;
+        private readonly IStorageService _storageService;
+
+        public UserService(UserManager<User> userManager, IMapper mapper, ICurrentUser currentUser, 
+            IStorageService storageService)
+        {
+            _userManager = userManager;
+            _mapper = mapper;
+            _currentUser = currentUser;
+            _storageService = storageService;
+        }
+
+        public async Task<UserView> GetUserByEmail(string email)
+        {
+            if (await _userManager.FindByEmailAsync(email.Trim().Normalize()) is not { } user)
```

Comment:
Remove `normalize `and simplify code 
```suggestion
            var user = await _userManager.FindByEmailAsync(email.Trim());
            if (user is null)
```

Reply:
Corrected as you wrote